Author: Luca Boccassi <luca.boccassi@gmail.com>
Description: Fix broken uvmnext_isr_top_half() function definition
 From kernel 4.6-rc3 -Werror=incompatible-pointer-types is used, which flags
 a function definition in uvm/uvm_common.c which does not match a prototype.

--- a/nvidia-uvm/uvm_common.c
+++ b/nvidia-uvm/uvm_common.c
@@ -60,7 +60,7 @@
 {
 
 }
-static NV_STATUS uvmnext_isr_top_half(void)
+static NV_STATUS uvmnext_isr_top_half(UvmGpuUuid *pGpuUuidStruct)
 {
     return NV_ERR_NO_INTR_PENDING;
 }
